hokusai/VERSION

```diff
@@ -0,0 +1 @@
+999.999.999
```

Comment:
not 0.0.0 b/c [some artsy projects](https://github.com/artsy/gravity/blob/caf28fd0345e50ae928d66404e3b15fc0b902707/hokusai/config.yml#L5) have min Hokusai version requirements.

Reply:
it's just a placeholder

Reply:
Minor: could be worth adding this context as a comment as it is non-obvious.

Reply:
True. But a comment would complicate [the code that reads the file](https://github.com/artsy/hokusai/pull/389/files#diff-c289bc883a0cbdc59ef79744e976404261024f587ca9a8b3af08e1396da946a8R6). How about [adding the info to README](https://github.com/artsy/hokusai/pull/389/commits/e59d52944a88e7a82d62968e48e0488b623bfb28)?

Reply:
That works!